### PR TITLE
Patch 1 

### PR DESCRIPTION
**Pull request by K-developer**
Changes

### CODE REVIEW

**Comment by K-developer** [comment]
# Purpose
Request for merging

- [x] Is it important
- [x] is it setup

**Comment by DoctorDib** [comment]
yes

File: false/source/CKSM.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CONST CHECKSUM = ()=> {
+	    if(++[+[]][+[]]+[+[]]  ? ++[+[]][+[]] : ++[+[]][+[]]*[+[]]) return ++[+[]][+[]]
```
Comment (DoctorDib) [inline false/source/CKSM.js:2]: yes

**Review by DoctorDib — COMMENTED** [review]
yes

**Pull request merged**